src/test/java/org/springframework/integration/aws/s3/config/xml/AmazonS3OutboundChannelAdapterParserTests.java

```diff
@@ -155,4 +172,13 @@ public String generateFileName(Message<?> message) {
 			return null;
 		}
 	}
-}
+
+	public static class FooAdvice extends AbstractRequestHandlerAdvice {
+
+		@Override
+		protected Object doInvoke(ExecutionCallback callback, Object target, Message<?> message) throws Exception {
+			adviceCalled++;
+			return callback.execute();
+		}
+	}
+}
```

Comment:
Sure and Thanks a lot Artem. It was a great learning experience.

I will add a new line manually at the end of file.